lib/Controller/CallController.php

```diff
@@ -127,6 +127,29 @@ public function joinCall(?int $flags = null, ?int $forcePermissions = null): Dat
 		return new DataResponse();
 	}
 
+	/**
+	 * @PublicPage
+	 * @RequireModeratorParticipant
+	 *
+	 * @param int $attendeeId
+	 * @return DataResponse
+	 */
+	public function ringAttendee(int $attendeeId): DataResponse {
+		if ($this->room->getCallFlag() === Participant::FLAG_DISCONNECTED) {
+			return new DataResponse([], Http::STATUS_BAD_REQUEST);
```

Comment:
I don't think it matters much, as in all the cases the UI option should not be displayed anymore. It can basically only happen in parallel executions when the UI is going to hide the option anyway soon.